.github/workflows/PublishNuget.yml

```
@@ -0,0 +1,35 @@
+      - name: Build
+        run: dotnet build source/Consolonia.sln -c Release --no-restore 
+  
+      - name: dotnet pack 
+        run: |
+          dotnet pack --no-build source/Consolonia.sln -c Release -o packages --include-symbols --property WarningLevel=0
+
+      - name: Publish NuGet and symbols
+        id: nuget-push
+        uses: edumserrano/nuget-push@v1
+        with:
+          api-key: '${{ secrets.NUGET_KEY }}' 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)